app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java

```diff
@@ -285,6 +286,7 @@ private void setupViewPager( ViewPager viewPager )
 			{
 				adapterResult.addFragment( new ProductPhotosFragment(), "Product Photos" );
 			}
+			adapterResult.addFragment( new IngredientsAnalysisProductFragment(), "Ingredient Analysis" );
```

Comment:
Can you extract the string (and also extract Product Photos, while you're at it) ?

Reply:
yeah I will do it.